src/Search.js

```diff
@@ -0,0 +1,84 @@
+import './Search.css';
+import axios from 'axios';
+import { useEffect, useState } from "react";
+import GoBack from './GoBack';
+
+const Search = () => {
+
+    //call useState hook to hold the array back from my api - destructure variable and function 
+    const [heroData, setHeroData] = useState([]); //expecting an array of objects, so I initialize with empty one
+    //character name coming from user input
+    const [userInput, setUserInput] = useState('');
+
+  
+  
+    useEffect(() => {
+  
+      if(userInput) {
+         axios({
+        baseURL: 'https://gateway.marvel.com/',
+        url: 'v1/public/characters',
+        method: 'GET',
+        params: {
+          ts: 'thesoer',
+          apikey: 'f6a569770dbc42c8e49153f86a614843',
+          hash: '2b386c7f05318a6e66fed6f3416c4b2d',
+          nameStartsWith: `${userInput}`,
+          limit: 15
+        }
+      }).then((apiData) => {
+  
+        setHeroData(apiData.data.data.results)
+      })
+      }
+     
+    }, [userInput])
+
+    //this event will handle the user typing in the input
+  //when the user types in the input box: 
+    const handleInputChange = (event) => {
+      const value = event.target.value;       
+      setUserInput(value);
+    }
+
+    return (
+        <div className='searchContainer' >
+          <h1 className='searchTitle' >Search</h1>
+          <div className='characterSearch' >
```

Comment:
May want to add some instructions for the user to indicate what they should search or the purpose of the search.